pkg/features/kube_features.go

```diff
@@ -300,6 +306,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	PersistentLocalVolumes:                      {Default: true, PreRelease: utilfeature.Beta},
 	LocalStorageCapacityIsolation:               {Default: true, PreRelease: utilfeature.Beta},
 	HugePages:                                   {Default: true, PreRelease: utilfeature.Beta},
+	Sysctls:                                     {Default: true, PreRelease: utilfeature.Alpha},
```

Comment:
I think we are wanting to move this to beta as part of the transition since we know that people are using them and are ok with the interface.

Reply:
I was not able to find more detailed documentation about what it means to promote a field/feature from alpha to beta. Though, after some discussion with folks and reading various PRs promoting from alpha to beta, there are two remaining things to do:

* change `PreRelease` to `utilfeature.Beta`
* rename `--experimental-allowed-unsafe-sysctls` to `--allowed-unsafe-sysctls`